packages/web3/test/e2e/sepolia/get_past_logs.test.ts

```diff
@@ -56,24 +57,85 @@ describe(`${getSystemTestBackend()} tests - getPastLogs`, () => {
 
 	it.each(
 		toAllVariants<{
-			format: string;
+			byteFormat: string;
+			numberFormat: string;
 		}>({
-			format: Object.values(FMT_BYTES),
+			byteFormat: Object.values(FMT_BYTES),
+			numberFormat: Object.values(FMT_NUMBER),
 		}),
-	)('should getPastLogs for deployed contract', async ({ format }) => {
+	)('should getPastLogs for deployed contract', async ({ byteFormat, numberFormat }) => {
 		const result = await web3.eth.getPastLogs(
 			{
 				fromBlock: 'earliest',
 				toBlock: 'latest',
 				address: '0xEdFd52255571b4a9A9d4445989E39f5c14Ff0447',
```

Comment:
what is the address? If it is the same for all mainnet tests it will be great to have it as a constant. what do you think?

Reply:
The address isn't the same on mainnet, so I [recommend](https://github.com/web3/web3.js/pull/6101#discussion_r1197133330) adding it to the `.secrets` file

Reply:
updated, thank you!